lib/signal-handler.c

```diff
@@ -84,12 +84,16 @@ signal_handler_exec_external_handler(gint signum)
 static int
 _original_sigaction(int signum, const struct sigaction *act, struct sigaction *oldact)
 {
+#ifdef __NetBSD__
+  __libc_sigaction14(signum, act, oldact);
```

Comment:
The original sigaction result should be returned.

```suggestion
  return __libc_sigaction14(signum, act, oldact);
```